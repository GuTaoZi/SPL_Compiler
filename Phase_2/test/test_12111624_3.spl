int foo1(){
    return 1;
    foo1();
}

int foo2(){
    foo1();
    foo2(1,2);
}

int main(){
    foo2(a);
}