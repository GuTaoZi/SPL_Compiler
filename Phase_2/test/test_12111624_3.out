Error type 9 at Line 8: parameters missmatch for function: 'foo2'
Error type 1 at Line 12: variable 'a' is used without a definition.
Error type 9 at Line 12: parameters missmatch for function: 'foo2'
